### ISSUE TITLE
Vue 3 support

### ISSUE THREAD

**Issue by milewski** [BinarCode/vue2-transitions#42]
Doesnt work with vue 3 `Uncaught TypeError: Cannot read property '_c' of undefined`

**Comment by cristijora** [comment]
Thanks for the issue @milewski Planning to add it in the upcoming days!

**Comment by bci24** [comment]
any updates ? :)

**Comment by bibibiu2017** [comment]
any updates on this?

**Comment by JenuelDev** [comment]
any update on this? thanks thanks :)

**Comment by JenuelDev** [comment]
@milewski have you tried importing it as a component instead of importing it in Vue.use.

like this?
```
<template>
   <fade-transition>
      <div class="box" v-show="show">
        <p>Fade transition</p>
      </div>
    </fade-transition>
</template>

<script>
import {FadeTransition} from 'vue2-transitions'

export default {
  components: {
    FadeTransition
  }
}
</script>
```

**Comment by milewski** [comment]
@BroJenuel Yes I have never used this library as `Vue.use` have always imported as a component like in your example

**Comment by JenuelDev** [comment]
@milewski oh ok, I'm about to use it in vue 3 ecosystem. thanks, thanks, ill be using vue 2 then. I'll upgrade it later if this supports vue 3

**Comment by MorevM** [comment]
This repository looks abandoned, so I got inspired and wrote a [new package for working with transitions](https://github.com/MorevM/vue-transitions).
It supports Vue 3 and much more :)

**Comment by 50rayn** [comment]
Hello Vue3 fans.

I created a module, basing on this one, after using it for around 4 years. Working a very big project in a product company, we decided to migrate the whole project to Vue3. As expected, where some modules which are no longer maintained or are badly added Vue3 support. So, my motivation was to create a module which will easily replace this one, but in a Vue3 project. I tried to use the already existing module [vue3-transitions](https://www.npmjs.com/package/vue3-transitions), but it didn't have all the animation that I needed, the bundle size was too big for 2 animations only and  too many changes I should have been done in case of it's integration. The main challenge to me was to keep the same props and the same events and try to optimize it more. My module is not very popular, but seems works as fine as worked this one for me for years. The main reason why i wrote this module, is for my personal use. I hope it will help someone too.

Transition Module: [vue3-sfc-transitions](https://www.npmjs.com/package/vue3-sfc-transitions)

The another module, that I mentioned above is called [vue3-drr-grid-layout](https://www.npmjs.com/package/vue3-drr-grid-layout). Yeah, strange naming 😄, agree. It was originally inspired from [vue-grid-layout](https://www.npmjs.com/package/vue-grid-layout). This module has the same story as the transition one. The biggest improvement was making bundle size smaller, from 7.12 MB to 321 kB.

Grid Module: [vue3-drr-grid-layout](https://www.npmjs.com/package/vue3-drr-grid-layout)

Soon, I'll try to create alternative module for Highcharts wrapper. There already exists 2 modules, [vue-highcharts](https://www.npmjs.com/package/vue-highcharts) and the official [highcharts-vue](https://www.npmjs.com/package/highcharts-vue). The main reason why I'll create the alternative is that none of them supports [SparkLine](https://www.highcharts.com/demo/sparkline).

Sorry for bothering you and thanks in advance.

Special thanks to @cristijora for his module that gave me opportunity to improve the app and motivate me to create my own. _Salut @cristijora de peste Prut_.
